target/linux/ramips/dts/GB-PC1.dts

```diff
@@ -0,0 +1,157 @@
+/dts-v1/;
+
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	model = "GB-PC1";
```

Comment:
I checked the dts(i) files for MT7621 devices.
I did not find a single example of a "compatible string"
Would you like me to add one for every MT7621 device?